students/Josh_HOff/lesson06/test_mailroom4.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+import mailroom4
+from mailroom4 import check_input
+from mailroom4 import check_donation
+from mailroom4 import create_report
+from mailroom4 import letters_to_everyone
+
+class MyTests(unittest.TestCase):
+
+    def setUp(self):
+        pass
+    def test_1(self):
+        self.assertFalse(check_input('quit'))
```

Comment:
should have one line space between functions in a class

Reply:
Got it; fixed